app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class RentsController < ApiController
+  def index
+    render json: Rent.all.page(params[:page])
+  end
+
+  def create
+    rent = Rent.new(rent_params)
+    if rent.save
+      render json: rent
+    elsif rent.book_id.nil? || rent.user_id.nil? || rent.to.nil? || rent.from.nil?
```

Comment:
You're validating the model twice, since `rent.save` will fail if any validation on the model fails.